web/templates/index.html

```diff
@@ -2,18 +2,39 @@
 {% load staticfiles %}
 
 {% block content %}
-<div id="main-content"></div>
+
 <div class="container">
+  <div class="hero-unit">
+    <h1>Welcome to 10,000 Hours</h1>
+    {% if user.is_authenticated %} 
+      <p>Welcome, {{ user.username }}. You are logged in. <a href="{% url 'logout' %}">[Sign Out]</a> </p>
+
+    {% else %}
+
+      <p>Welcome, new user. Please log in or create an account.</p>
+      <p><a href="{% url "create-new-user" %}" class="btn btn-primary btn-large">Register</a>
+         <a href="{% url "login" %}" class="btn btn-success btn-large">Login</a>
+      </p>
+    {% endif %}
+
+  </div>    
 
-  <h1>Welcome to 10,000 Hours </h1> 
   {% if user.is_authenticated %}
-    <p>Welcome, {{ user.username }}. You are logged in. </p>
-    <h3>Your skills</h3>
-    <ul>
-    {% for skill in skills %}
-    <li><a href="/skills/{{ skill.id }}/">{{ skill.name }}</a></li>
-    {% endfor %}
-  </ul>
+
```

Comment:
I wouldn't put a newline between these if statements and their subsequent blocks.